src/graphql/typeDefs/dataTypes/enums/sessionStatus.graphql

```diff
@@ -0,0 +1,13 @@
+enum SessionStatus {
```

Comment:
We're not really there yet, but as things start to ramp up, I could see an IN_PROGRESS flag to where you could query what's currently runnning. We would be exposed to data being borked up if there were errors.. but interesting idea.